src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/UnnecessaryParenthesesCheck.java

```diff
@@ -571,19 +575,22 @@ else if (ast.getParent().getType() == TokenTypes.LITERAL_RETURN) {
 
     /**
      * Checks if conditional, relational, unary and postfix operators
-     * in expressions are surrounded by parentheses.
+     * in expressions are surrounded by unnecessary parentheses.
      *
      * @param ast the {@code DetailAST} to check if it is surrounded by
-     *        parentheses.
+     *        unnecessary parentheses.
      * @return {@code true} if the expression is surrounded by
-     *         parentheses.
+     *         unnecessary parentheses.
      */
-    private static boolean checkAroundOperators(DetailAST ast) {
+    private static boolean unnecessaryParenAroundOperators(DetailAST ast) {
```

Comment:
The name was a bit misleading as well as the Javadoc, edited them to make more sense.